joomla/joomla-cms
# [4.0][RTL] CSS logical properties initial

Pull Request for Issue #28546 .

Also removed some redundant rules ( #28586 & #28585).

### Summary of Changes
Implements some logical properties to reduce on RTL CSS.

### Testing Instructions
Apply this patch and run `node build.js --compile-css` to update SCSS. Check RTL styling.
